src/Plugin/AdvancedQueue/JobType/SendDigitalPost.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\os2forms_digital_post\Plugin\AdvancedQueue\JobType;
+
+use Drupal\advancedqueue\Job;
+use Drupal\advancedqueue\JobResult;
+use Drupal\advancedqueue\Plugin\AdvancedQueue\JobType\JobTypeBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\os2forms_digital_post\Helper\WebformHelper;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Archive document job.
```

Comment:
I also forgot to remove hints of me stealing this from somebody else when I wrote a handler …

Reply:
Changed comments